webapp/channels/src/components/drafts/drafts.tsx

```diff
@@ -65,32 +81,38 @@ function Drafts({
                     defaultMessage: 'Any messages you\'ve started will show here',
                 })}
             />
-            <div className='Drafts__main'>
-                {drafts.map((d) => (
-                    <DraftRow
-                        key={d.key}
-                        displayName={displayName}
-                        draft={d}
-                        isRemote={draftRemotes?.[d.key]}
+
+            <Tabs
+                id='draft_tabs'
+                activeKey={activeTab}
+                mountOnEnter={true}
+                unmountOnExit={false}
```

Comment:
We discussed in feature channel that we would not unmount and cause re-rendering a large list when user switches back. It will get unmount when you switch out of the drafts section.